alpha/apps/kaltura/modules/extwidget/actions/thumbnailAction.class.php

```diff
@@ -570,12 +570,12 @@ protected function getLastModified(entry $entry, $isCapturing)
 
 	private function getDimensions()
 	{
-		$width = $this->getFloatRequestParameter("width", self::DEFAULT_DIMENSION, -1, 10000);
-		$height = $this->getFloatRequestParameter("height", self::DEFAULT_DIMENSION, -1, 10000);
+		$width = $this->getIntRequestParameter('width', self::DEFAULT_DIMENSION, -1, 10000);
```

Comment:
the -1 and 10000 can be added as default min and max consts

Reply:
True but there different default min and max in this class so I didn't want to touch this part since its legacy code